src/session.ts

```diff
@@ -57,6 +60,7 @@ export const createState = (initial: State) => {
           connection: Connection.Connected,
           session: { address, tokenBalance, tx: null }
         });
+        window.localStorage.setItem("session", JSON.stringify({ address, tokenBalance }));
```

Comment:
Might be worth setting `tx: null` here, otherwise it'll be `undefined` when read back out, which is not well typed.